qual/src/main/java/com/virtuslab/qual/internal/NotUiThreadSafe.java

```diff
@@ -0,0 +1,3 @@
+package com.virtuslab.qual.internal;
```

Comment:
Hmm I meant `internal` package as a place for stuff that's not meant to be used outside of `qual` subproject... kinda "module-private" stuff :)

Reply:
I think this one can go to `com.virtuslab.qual.guieffect` or so

Reply:
Maybe add package-info.java to `internal` package to make it clear